src/datadog/metrics.cpp

```diff
@@ -5,24 +5,24 @@
 namespace datadog {
 namespace tracing {
 
-Metric::Metric(const std::string name, std::string type,
-               const std::vector<std::string> tags, bool common)
+Metric::Metric(std::string name, std::string type,
+               std::vector<std::string> tags, bool common)
     : name_(name), type_(type), tags_(tags), common_(common) {}
```

Comment:
Since we're accepting these parameters by value, might as well `std::move` them into these initializers.

Alternatively, use `const T&` instead.

Alternatively, use `const vector&` and `StringView`.

Alternatively, use `std::move(vector)` and `StringView`.

It's true that this code runs on tracer startup only, so it doesn't matter. But, style.

Reply:
You might also just leave it like this. Less ink.

Reply:
Now that I'm looking at the individual commits again, I wonder if the original `const` that you had on these parameters was a typo where you were missing the `&`. Anyway, figure out what you want the signatures and initializers to look like, based on how the constructor is called.